ajax_with_jquery_exercise/index.js

```diff
@@ -0,0 +1,62 @@
+$(document).ready(function(){
+	//variables
+	var auth_token;
+
+	//imeadiat calls
+	getTop().then(function(data){
+		debugger 
+	})
+
+	//click functions
+
+	//functions
+	function getTop(){
+		return $.get('https://hacker-news.firebaseio.com/v0/'
+			+ 'topstories.json?print=pretty')
+		.then(function(topstories){
+			console.log(topstories);
+			return Promise.all(topstories.slice(0,50).map(function(v,i){
+				return $.get(
+					`https://hacker-news.firebaseio.com/v0/' 
+					+ 'item/${topstories[i]}.json?print=pretty`)
```

Comment:
It looks like you mixed single quotes and back tics.  I'm not sure what that does actually.